### PR TITLE
X11 apps drop 32bit

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

**Review by pyhalov — APPROVED** [review]

File: components/x11/cmap_alloc/Makefile [hunk 1]
```diff
@@ -13,45 +13,35 @@
...
 COMPONENT_CLASSIFICATION= System/X11
-COMPONENT_SUMMARY=  			cmap_alloc - colormap allocation utility
-COMPONENT_SRC=      			src
-COMPONENT_PROJECT_URL = https://hg.java.net/hg/solaris-x11~x-s12-clone
-COMPONENT_LICENSE=      Oracle
-COMPONENT_LICENSE_FILE= cmap_alloc.license
+COMPONENT_SUMMARY=    cmap_alloc - colormap allocation utility
+COMPONENT_SRC=        src
+COMPONENT_PROJECT_URL=https://hg.java.net/hg/solaris-x11~x-s12-clone
+COMPONENT_LICENSE=    Oracle
```
Comment (AndWac) [inline components/x11/cmap_alloc/Makefile:29]: What about "TEST_TARGET = $(NO_TESTS)" here?

File: components/x11/xcolor/Makefile [hunk 2]
```diff
@@ -13,45 +13,36 @@
...
-COMPONENT_LICENSE_FILE= xcolor.license
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/justmake.mk
-include $(WS_TOP)/make-rules/ips.mk
+TEST_TARGET=$(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+PATH=$(USRBINDIR)
```
Comment (AndWac) [inline components/x11/xcolor/Makefile]: Is /usr/gnu/bin not needed anymore?

Comment (pyhalov) [reply components/x11/xcolor/Makefile]: Just remove PATH= line?

File: components/x11/cmap_alloc/Makefile (hunk 1 above)
Comment (alarcher) [reply components/x11/cmap_alloc/Makefile:29]: Thank you, good catch!

File: components/x11/xcolor/Makefile (hunk 2 above)
Comment (alarcher) [reply components/x11/xcolor/Makefile]: Yes it does not seem to be used, I just removed it. Thank you :)

Comment (AndWac) [reply components/x11/xcolor/Makefile]: Hm, I meant it the other way around: why did you leave out /usr/gnu/bin from PATH :)

Comment (alarcher) [reply components/x11/xcolor/Makefile]: OK OK, it does not seems to be used but re-added it as before anyways.

**Pull request merged**